### DISCUSSION TITLE
How to select/delete multiple items?

### DISCUSSION THREAD

**Discussion by Drazzilb08** [SergiX44/XBackBone#481] [Q&A]
Is there currently a way to delete multiple items at the same time?
I see this button
![img](https://i.imgur.com/s4pez8w.png)

But it is always grayed out no matter what I select

Thanks for the reply.

**Comment by cloonix** [comment]
You can right click on file names to select them. 

Had trouble with this myself. 

AN option to select all items would be nice though.
